qa/rpc-tests/wallet_listreceived.py

```diff
@@ -309,7 +309,7 @@ def test_received_orchard(self, height):
         opid = self.nodes[1].z_sendmany(taddr, [
             {'address': uao, 'amount': 1, 'memo': my_memo},
             {'address': uaso, 'amount': 2},
-        ], 1, 0, 'AllowRevealedSenders')
+        ], 1, 0, 'AllowFullyTransparent')
```

Comment:
This is effectively a bugfix, because we now require a weaker privacy policy to perform the same operation. It's arguably up to interpretation, as there are no explicit transparent recipients, but also because the wallet doesn't send change to the same transparent address, there is an implicit transparent recipient that happens to be the same as the sender.

In any case, we should note this change in the release notes.